Cryptomator/AppDelegate.swift

```diff
@@ -130,11 +131,24 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 	private func setupIAP() {
 		#if ALWAYS_PREMIUM
 		DDLogDebug("Always activated premium")
-		GlobalFullVersionChecker.default = AlwaysActivatedPremium.default
 		CryptomatorUserDefaults.shared.fullVersionUnlocked = true
 		#else
 		DDLogDebug("Freemium version")
-		GlobalFullVersionChecker.default = UserDefaultsFullVersionChecker.default
+		#endif
+	}
+}
+
+/**
+ Define the liveValue in the main target since compilation flags do not work on Swift Package Manager level.
+ Be aware that it is needed to set the default value once per app launch (+ also when launching the FileProviderExtension).
+ */
+extension FullVersionCheckerKey: DependencyKey {
+	public static var liveValue: FullVersionChecker {
+		#if ALWAYS_PREMIUM
+		GlobalFullVersionChecker.default = AlwaysActivatedPremium.default
+		CryptomatorUserDefaults.shared.fullVersionUnlocked = true
```

Comment:
I believe you've deleted `GlobalFullVersionChecker` and this case doesn't have a return value. Is it possible that the `ALWAYS_PREMIUM` case is not quite finished? 😁

Reply:
Ah good catch! I fixed it by returning `AlwaysActivatedPremium.default` 🙈